styles.css

```diff
@@ -221,25 +222,30 @@ li {
   padding: 12px;
 }
 
-/* Work section*/
+/* Work section */
 
 .Works {
-  display: grid;
+  display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 120px 0px;
+  padding: 120px 0.001px;
   background: #fdfdfd;
   position: absolute;
   width: 100%;
   height: 3658.4px;
-  left: 0px;top: 682px;
-  font-family: Crete Round;
+  left: 0.001px;
+  top: 682px;
+
+  /* decoration */
```

Comment:
Your code explains by itself what it do.
Comments are helpful on a development environment to provide support to your partner.
Once that you are requesting a code review it's recommended to have a clean code without comments at least linters can keep.